### ISSUE TITLE
Converting to Java

### ISSUE THREAD

**Issue by akideru** [akideru/Games#1]
phff Switching to Nukkit, PocketMine es bad

**Comment by EpicSteve33** [comment]
I would help you with this but I do not know java or the nukkit api ;-;

**Comment by akideru** [comment]
@EpicSteve33 I'm just guessing and checking lol

**Comment by EpicSteve33** [comment]
so far the only thing I made in java was a plugin that says hello when a player joins I dk if it works never tested it lol

**Comment by akideru** [comment]
have 100 errors

**Comment by EpicSteve33** [comment]
lol Idk how to compile it into a jar file

**Comment by akideru** [comment]
just use DevTools

**Comment by EpicSteve33** [comment]
plus ive been busy playing on my modded server

**Comment by akideru** [comment]
games don't count for being busy

**Comment by EpicSteve33** [comment]
well I made a mod for my server so i have been busy working on that

**Comment by EpicSteve33** [comment]
I used mcreator :0

**Comment by akideru** [comment]
Been busy reading a manga

**Comment by akideru** [comment]
https://i.imgur.com/NzM1egY.png Tell Mexi to buy me this

**Comment by akideru** [comment]
@EpicSteve33 Gave up on Java.. Just help me Code New Games into this Working on Gravity rn

**Comment by orlando092** [comment]
You fucking suck, that is my code. Piece of dirt, piece of fertilizer, fucking cunt, moldy apple, nigger faggot

**Comment by orlando092** [comment]
Original code: https://github.com/TeamPocketPHP/UHC-Simulator-Automatico

**Comment by orlando092** [comment]
Bitch

**Comment by orlando092** [comment]
Female dog

**Comment by akideru** [comment]
@orlando092 yeah sorry, I'm not doing anything with it other than using it for my server

**Comment by orlando092** [comment]
Well give me credits

**Comment by EpicSteve33** [comment]
@orlando092 suck a dick, your plugin sucks
